styles/base/_root.scss

```diff
@@ -0,0 +1,8 @@
+html {
```

Comment:
You may have noticed that these new SCSS files all begin with an underscore. That's because the compiler will create a new .css file for every SCSS file it finds that doesn't begin with an underscore

Since we only want one request, the main style.scss file is the only SCSS file in the project that doesn;t begin with an underscore!!!